runtime/src/main/jni/CallbackHandlers.cpp

```diff
@@ -317,8 +317,7 @@ void CallbackHandlers::CallJavaMethod(const Local<Object>& caller, const string&
         callerJavaObject = objectManager->GetJavaObjectByJsObject(caller);
         if (callerJavaObject.IsNull()) {
             stringstream ss;
-            ss << "No java object found on which to call \"" << methodName <<
-               "\" method. It is possible your Javascript object is not linked with the corresponding Java class. Try passing context(this) to the constructor function.";
+            ss << "Failed calling " << methodName << " on a " << className << " instance. The JavaScript instance no longer has available Java instance counterpart.";
```

Comment:
It is better, and it's part of a bigger plan to check not only method and constructor calls but all possible calls with JNI, but we haven't gotten around to it. I am four thumbs up for this effort!